index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7967e81f77.js" crossorigin="anonymous"></script>
+    <title>Portfolio</title>
+</head>
+<body>
+   <div class="container">
+       <header>
+           <nav class="navItems">
+            <h1>Welcome</h1>
```

Comment:
- [ ] Please wrap this section in an `a` tag as this logo will be used as a link to the home page
- [ ] You should consider changing that `h1` to something hx below as you already have `h1` for your title in the headline section